src/generator/synth.ts

```diff
@@ -82,7 +82,11 @@ export async function synth(options: SynthOptions = {}) {
     if (statusFiles.filter(x => x.startsWith(`discovery/${dir}-`)).length > 0) {
       await execa('git', ['add', `discovery/${dir}-*`]);
     }
-    const commitParams = ['commit', '-m', title];
+    // Write commit message to file, since it might be large enough to
+    // cause spawn E2BIG in CI/CD:
+    const message = changelog ? `${title}\n\n${changelog}` : title;
+    fs.writeFileSync('message.txt', message, 'utf8');
```

Comment:
Should we remove it later, or is it happening in some temporary place?

Reply:
maybe remove it immediately after?

Reply:
unlink now.